### DISCUSSION TITLE
Command Alert Error

### DISCUSSION THREAD

**Discussion by ozgursuder** [jertel/elastalert2#930] [Q&A] [answered]
Error while running alert command: Error while running command sshpass -p 'password' ssh user@pc.local@10.0.0.1 stop-process -id 11388 -Force: [Errno 2] No such file or directory: "sshpass -p 'password' ssh user@pc.local@10.0.0.1 stop-process -id 11388 -Force"

Hello

All alerts are working fine. but I only get the following error in the command alert section.

where am i doing wrong

Can you help with this

**Comment by jertel** [comment, marked as answer]
As mentioned in the [docs](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#command), the Command alerter uses a list of command line args, not a single string. You haven't provided logs or config files to reproduce your experience so I am assuming that you attempted something like this:
```
command: sshpass -p 'password' ssh user@xxx stop-process -id 11388 -Force
```
But it should be in list format:
```
command: 
  - sshpass
  - -p
  - 'password'
  - ssh
  - user@xxx
  - stop-process
  - -id
  - yyy
  - -Force
```
Also, double-check your user@host arg, as it looks like you could have a mistake there.

**Comment by ozgursuder** [comment]
thanks 

My problem was solved with your information.
